### ISSUE TITLE
force calling with positions which have not been identified at first run

### ISSUE THREAD

**Issue by quanc1989** [fritzsedlazeck/Sniffles#162]
Hi Fritz,
I try to do sv calling with three methods (sniffles, nanosv, svim) for the same sample (bam aligned by ngmlr), and use SURVIVOR to merge these breakpoints. Then I use sniffles to force calling  these positions. Here some questions emerge:
1. positions which have not been identified at first run is confirmed at second run. There are even some SVs with RE>10 and GT=1/1
2. small number of  positions in the merged vcf are missing in the final result.
3. Instead of force calling, is there any methods to genotype these SVs for Nanopore sequencing data? 

Thanks!

**Comment by fritzsedlazeck** [comment]
Hi Achilles,
this mode is still in a beta version. So thanks for the feedback. 

@1 yes the genotyping does not include any automated filtering. 
@2 please show/ send me examples.
@3 I am not aware of any. 
Thanks
Fritz

**Comment by achilles1989** [comment]
Thanks for your quick reply!
I think I haven't clearly stated my problem. I know the genotyping (force calling) does not include any automated filtering, What I don't understand is why these SVs are not recognized by sniffles at the first run. According to the force-calling result, these SVs seem obvious and significant. And whether it makes sense to do force-calling with positions which sniffles didn't find originally, because no genotypers are available for Nanopore sequencing data now.

**Comment by achilles1989** [comment]
I submit examples to OneDrive,
https://1drv.ms/u/s!Akst2lst1mnzgp4d4QCTw5KMoh0o-w?e=HCUtn6

It contains three vcf files,
1. merge.ont.sorted.reheader.vcf, which is merged across 30 samples by SURVIVOR and includes 45008 Sites.
`SURVIVOR merge merge.files 1000 1 -1 0 0 50 merge.ont.sorted.reheader.vcf`

2. CQ082.genotyped.vcf, which is genotyped by force-calling and includes 44780 Sites.
`sniffles -s 5 -t 80 -m CQ082.bam -v CQ082.genotyped.vcf --genotype --report_BND -n -1 -l 50 --Ivcf merge.ont.sorted.reheader.vcf`
3. CQ085.genotyped.vcf, another example and also includes 44780 Sites.

There are always 45008-44780=222 Sites missed in the force-calling results. I think these sites have been merged by some adjacent sites, but how can this happen after I already merged them at first step by SURVIVOR with a maximal distance of 1000bp?
